9.2/run-postgresql.sh

```diff
@@ -33,5 +33,9 @@ if [ "$1" = "postgres" -a ! -f "$PGDATA/postgresql.conf" ]; then
   initialize_database
 fi
 
+pg_ctl -w start
+set_passwords
+pg_ctl stop
+
```

Comment:
Sadly there are two code flows for postgresql-master (replication example) and standalone.
We can look into refactoring this later.